### DISCUSSION TITLE
2.1.x versions stuck on starting services...full node

### DISCUSSION THREAD

**Discussion by BrandtH22** [Chia-Network/chia-blockchain#18178] [Support]
2.01 is working and 2.0.0 and 1.8.1 but 2.1.0  2.1.1 and 2.3.1 doesn't work. They says they are starting the service of the full node and I have waited the whole night so it is never happening. I tried going back and forth in the versions and it seems the new ones aren't working.
I am running win10 could upgrading to win11 help ?
I have seen that others have had the same problem in the past with other versions and I think I have had this problem or similar too in much earlier versions.

_Originally posted by @svenove7 in https://github.com/Chia-Network/chia-blockchain/discussions/18097#discussioncomment-9768108_

**Comment by BrandtH22** [comment]
Hey @svenove7 , as a heads up we can generally provide more thorough and timely support in our discord server (https://discord.gg/chia). 

As for the stuck on starting services, there are two reasons this occurs either that the blockchain db needs to be updated from v1 to v2 or your system has more than 61 cores. Both of these have known solutions listed below:

NOTE - make sure to fully stop chia before making any changes to the needed files.

To verify if the blockchain db needs to be updated navigate to `~/.chia/mainnet/db/` and look at the .sqlite file, if it has V1 in the title then we need to update and you can proceed with the steps outlined here:
https://docs.chia.net/faq/#how-do-i-upgrade-my-version-1-blockchain-database-to-version-2

If this is not the case then I would look at your log file located at `~/.chia/mainnet/log/debug.log` for a line that indicates:
`ValueError: max_workers must be <= 61`

If this line is present then you will need to navigate to the config file at `~/.chia/mainnet/config/config.yaml` and in the full_node section update the parameter `reserved_cores` to 10.

If neither of these are the issue let us know and we can further debug.

**Reply by svenove7** [reply]
Thanks !
I needed to update to version 2. A lot of steps but now I could start the newest version almost immediatly.

It is ok to delete blockchain_v1_mainnet.sqlite now right because I have the version2 now ?
BR
me

**Reply by BrandtH22** [reply]
Hey, yes that is correct you can remove the v1 db now that the v2 is in place 👍

**Comment by BrandtH22** [comment]
Since this is now resolved I am going to close this discussion but do not hesitate to reopen this ticket or open a new one if other questions arise.
